src/main/java/uk/gov/ons/census/action/model/entity/CaseToProcess.java

```diff
@@ -24,4 +24,6 @@
 
   private UUID batchId;
   private int batchQuantity;
+
+  @Column private Integer ceExpectedCapacity;
```

Comment:
bit of an oversight on my part... but the `batchId` and `batchQuantity` should also be marked with `@Column`